modules/solid_mechanics/test/tests/domain_integral_thermal/gold/j_integral_2d_out.e

```diff
@@ -1 +0,0 @@
-../../../../../tensor_mechanics/test/tests/domain_integral_thermal/gold/j_integral_2d_out.e
```

Comment:
It looks like these were all symlinks, and you replaced them with new gold files.  We should be able to still symlink to the tensor_mechanics results to make sure they produce the same results, shouldn't we?